articles/azure-app-configuration/faq.yml

```diff
@@ -146,15 +141,16 @@ sections:
 
       - question: Why can't I create an App Configuration store with the same name as one that I just deleted?
         answer: |
-          When a standard tier App Configuration store is deleted the name is reserved for seven days after deletion. It is possible to recreate a store with the same name before the reservation period has elapsed but it must be in the same subscription, resource group, and region as the original store. If the resource group that contained the store was deleted it will need to be recreated in the same subscription first before recreating the store inside of it.
-
-          The App Configuration store cannot be created in a different subscription/resource group until the retention period has elapsed. However, the resource can be recreated in the original subscription/resource group and then moved to a new subscription/resource group by following [these instructions](../azure-resource-manager/management/move-resource-group-and-subscription.md#use-the-portal).
+          All App Configuration stores in the standard tier have automatically enabled the [soft-delete](./concept-soft-delete) feature. When a standard tier App Configuration store is deleted, its name is reserved for the retention period. To recreate a store with the same name before the retention period expires, you need to [purge the soft-deleted store](./howto-recover-deleted-stores-in-azure-app-configuration#list-recover-or-purge-a-soft-deleted-app-configuration-store) first, provided the store doesn't have purge protection enabled.
```

Comment:
> provided the store doesn't have purge protection enabled

This leaves it a bit open ended. It may be worth it to confirm that if purge protection is enabled, the user must wait for the retention period to elapse.